wallet/src/main/java/com/android/identity_credential/wallet/MainActivity.kt

```diff
@@ -310,6 +372,7 @@ class MainActivity : ComponentActivity() {
                     credentialConfiguration.cardArt.size,
                     options)
                 val credentialName = credentialConfiguration.displayName
+                CREDENTIAL_ID = credentialId
```

Comment:
This is not really the correct way to get the currently selected credential. Also, we generally only use `ALL_CAPS_AND_UNDERSCORE` for constants, never for variables. Suggest to just add a TODO in `PresentationActivity` that if multiple credentials with the requested docType is available, prefer the one that might currently be actively selected in `MainActivity`'s pager.